pm-frontend/src/pages/SignIn.js

```diff
@@ -13,14 +14,25 @@ import Typography from "@mui/material/Typography";
 import Container from "@mui/material/Container";
 
 export default function SignIn() {
-  const handleSubmit = (event) => {
+  const apiEndpoint = "http://localhost:3001"
+
+  const handleSubmit = async (event) => {
     event.preventDefault();
     const data = new FormData(event.currentTarget);
     // eslint-disable-next-line no-console
     console.log({
       email: data.get("email"),
       password: data.get("password"),
     });
+    // example of how to hit the api endpoint.
+    // should use post here and send the email/password instead
+    try {
+      const res = await axios.get(apiEndpoint + '/')
+      alert(res.data.message)
+    } catch(error) {
+      console.log("Failed to login")
+      console.log(error)
+    }
```

Comment:
I just added this as a small demo on how to hit the endpoints. lmk if you want me to remove it before merging